stream_test.go

```diff
@@ -1,13 +1,15 @@
 package shoutcast
 
 import (
-	"fmt"
 	"io"
 	"math"
 	"net/http"
 	"net/http/httptest"
 	"reflect"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
Would you be very upset to not include these and keep it dependency free?

Reply:
No :)